### ISSUE TITLE
Note needed for Tmux 2.6

### ISSUE THREAD

**Issue by aleksandrs-ledovskis** [ChrisJohnsen/tmux-MacOSX-pasteboard#66]
Since https://github.com/tmux/tmux/commit/78352fdd328b611d935ba051288191b33278cc4a and [release 2.6](https://github.com/tmux/tmux/blob/8aaf86a6ead9852631342d0d2d526a7eaede15cf/CHANGES#L66) `reattach-to-user-namespace` trick doesn't seem to be needed.

```shell
$ echo "test" | pbcopy
$ pbpaste
test
```

**Comment by raejin** [comment]
You're right. I removed all the relevant copy vi mode binding and killed the tmux server. I can achieve the same result by only supplying the following binding:

```
bind-key -T copy-mode-vi v send -X begin-selection
bind-key -T copy-mode-vi y send-keys -X copy-pipe-and-cancel "pbcopy"
bind-key -T copy-mode-vi MouseDragEnd1Pane send-keys -X copy-pipe-and-cancel "pbcopy"
bind-key -T copy-mode-vi Enter send-keys -X copy-pipe-and-cancel "pbcopy"
```

I'm on macOS 10.13 with tmux 2.6

**Comment by lyuehh** [comment]
Still needed for some cli tools, like `trash`, or `brew home xx`

**Comment by aleksandrs-ledovskis** [comment]
@lyuehh 

I did install `trash` from homebrew and executed 

```shell
$ cd /tmp/
$ touch 123
$ trash 123
```
Afterwards, I verified that file "123" indeed was moved to macOS Trash.

Also, I tried `brew home` with `rust` and `python` arguments. Both commands succeeded in opening respective homepages in default browser.

So, appears working for me 🤷‍♂️

**Comment by lyuehh** [comment]
@aleksandrs-ledovskis 
On my computer , tmux was needed for `trash`, I don't known why ...

**Comment by chris-fa** [comment]
Can we add to the README some text that says in what tmux/macOS version this is not needed? :)

**Comment by luisgerhorst** [comment]
I'm not entirely sure if this is even related, but if you're using a recent iTerm there is a option "Applications in terminal may access clipboard" in the Preferences. When it is selected Tmux copies directly to the macOS system clipboard when using copy mode **without any configuration**. This even works over SSH. 

I can access `pbcopy` et al. (also `trash`) from within Tmux (version 2.7) on macOS 10.13.4.

**Comment by fdelacruz** [comment]
@raejin are those even needed anymore?? 
I'm running tmux 2.7 and I can copy/paste to and from, no problem like any other app in mac (mojave)

**Comment by aleksandrs-ledovskis** [comment]
@fdelacruz What method are you using? Aren't you mistaking text selection in terminal using mouse and pressing Cmd+C/Cmd+V vs. programmable copy & paste (as in `pbcopy`/`pbpaste`)

**Comment by fdelacruz** [comment]
@aleksandrs-ledovskis Hi, well, I'm not using the mouse. I'm simple selecting the text in vim'svisual mode and pasting outside of tmux. I can do the reverse from any app too. What gives??

**Comment by fdelacruz** [comment]
I'm using iTerm2 version 3 but there I haven't even enabled the "Applications in terminal may access clipboard" feature.

**Comment by aleksandrs-ledovskis** [comment]
@fdelacruz Thing is, @raejin's comment/snippet refers to Tmux `copy-mode` configuration & is not applicable/affecting applications launched within terminal multiplexer, such as Vim.

Regarding Vim and yank ending up in OS clipboard - could it be that you are yanking to  plus/`+` register (or have `unnamedplus` in `.vimrc`)?

**Comment by fdelacruz** [comment]
@aleksandrs-ledovskis yes, I have mine configured as ` set clipboard=unnamed`. 
Btw, i just copied from the top command into vim..

**Comment by der-flo** [comment]
I'm on tmux 2.9a and I need reattach-to-user-namespace to prevent this issue: https://github.com/macvim-dev/macvim/issues/622
